js/openfisca.js

```diff
@@ -121,7 +121,11 @@ function get(additionalParameters, callback) {
 /** Updates the displayed values.
 */
 function update() {
-	get({}, function(error, values, response) {
+	var today = new Date()
+
+	get({
+		contrat_de_travail_debut: today.getFullYear() + '-' + (today.getMonth() + 1),
```

Comment:
:+1: 
Sure that OpenFisca handles months on a single digit (without leading `0`)?